### ISSUE TITLE
windows?

### ISSUE THREAD

**Issue by su314ka** [TheRealLordFractal/Bip39Helper#3]
can be run on windows?
thanks!

**Comment by SatoshiNakamotoBitcoins** [comment]
@su314ka Real Lord has also a python repository you could apply with Windows:

https://github.com/TheRealLordFractal/Bip39Generator

**Comment by TheRealLordFractal** [comment]
This is correct, I can compile this for windows, but didnt think there was demand.. Make sure to update to my latest release.
